### ISSUE TITLE
Package as module for easy import in Observable

### ISSUE THREAD

**Issue by amitkaps** [vega/datalib#96]
To import `datalib` in Observable, requires it to be manually bundled first.  This is what is suggested by the [Module Require Debugger](https://beta.observablehq.com/@tmcw/module-require-debugger)
```
require('https://bundle.run/datalib@1.8.0')
```

Can we bundle it in a way that either of the following works?
```
require('datalib@1.8.0/src/index.js')
import('https://unpkg.com/datalib@1.8.0?module')
```

**Comment by domoritz** [comment]
Fixed in #97

**Comment by domoritz** [comment]
Example: https://beta.observablehq.com/@domoritz/hello-datalib
